<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureInput</class>
 <widget class="QWidget" name="ConfigureInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigureInput</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="2" column="0">
      <widget class="QGroupBox" name="faceButtons_3">
       <property name="title">
        <string>Shoulder Buttons</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>R:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonR">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>L:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonL">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>ZL:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonZL">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>ZR:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonZR">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="faceButtons_4">
       <property name="title">
        <string>Circle Pad</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="AnalogSetter" name="analogCirclePad" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="faceButtons_5">
       <property name="title">
        <string>C-Stick</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="AnalogSetter" name="analogCStick" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="faceButtons_2">
       <property name="title">
        <string>Directional Pad</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_34">
            <property name="text">
             <string>Up:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonDpadUp">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_35">
            <property name="text">
             <string>Down:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonDpadDown">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Left:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonDpadLeft">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>Right:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonDpadRight">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="faceButtons">
       <property name="title">
        <string>Face Buttons</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>A:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonA">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>B:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonB">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonX">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonY">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="faceButtons_6">
       <property name="title">
        <string>Misc.</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_25">
          <item>
           <widget class="QLabel" name="label_29">
            <property name="text">
             <string>Start:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonStart">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_26">
          <item>
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>Select:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonSelect">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_27">
          <item>
           <widget class="QLabel" name="label_31">
            <property name="text">
             <string>Home:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ButtonSetter" name="buttonHome">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRestoreDefaults">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Restore Defaults</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonSetter</class>
   <extends>QPushButton</extends>
   <header>configure_input_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnalogSetter</class>
   <extends>QWidget</extends>
   <header>configure_input_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
